### ISSUE TITLE
Custom map crash with themeless

### ISSUE THREAD

**Issue by GoogleCodeExporter** [lixun910/geoda#190]
```
1. Open NAT.shp
2. Change Current map type -> Create New Custom 
3. Select a variable HR60 

GeoDa will crash when click OK
```

Original issue reported on code.google.com by `lixun...@gmail.com` on 2 Jul 2015 at 9:55

**Comment by jkoschinsky** [comment]
If you follow these steps:

1. Open NAT.shp
2. Change Current map type -> Create New Custom 
3. Select a variable HR60 

GeoDa won't crash but there's still something wrong with the output (see screenshots below).

Let's remove "themeless" as an option from the category editor since that's a legend classification without any thresholds, which doesn't make sense here.

We could make natural breaks with 5 categories the default instead of user defined with 1 category as is currently the case.

![screen shot 2015-11-10 at 7 49 58 pm](https://cloud.githubusercontent.com/assets/13679743/11080685/5e881cfe-87e4-11e5-92ee-d287b143db75.png)
![screen shot 2015-11-10 at 7 49 06 pm](https://cloud.githubusercontent.com/assets/13679743/11080687/6065dc3c-87e4-11e5-8279-c2e01ff02edb.png)
![screen shot 2015-11-10 at 7 49 37 pm](https://cloud.githubusercontent.com/assets/13679743/11080688/64c537c8-87e4-11e5-99dc-7a7616645dd3.png)
![screen shot 2015-11-10 at 7 48 55 pm](https://cloud.githubusercontent.com/assets/13679743/11080691/6bad9936-87e4-11e5-8181-39811a268f6d.png)

**Comment by jkoschinsky** [comment]
Verified fix in GeoDa 1.7.87 (OSX Yosemite, 10.10.5 and Win 7 Professional (64-bit)).
